content/docs/user-guide/assets/fbx-settings/settings-physx-tab.md

```diff
@@ -129,13 +129,13 @@ A value of **0** performs no volume minimization and is recommended for most mes
 For meshes that have low vertex counts, or vertices that are distributed mainly along the edges, the algorithm can sometimes produce sub-optimal results where the primitive touches the edges of the mesh, but not the faces. In these cases, the fitting can be improved by increasing the **Volume Term Coefficient** so that the fitting algorithm shrinks the volume of the primitive while minimizing its deviation from the mesh. A **Volume Term Coefficient** that is too high can shrink the primitive collider so that it's occluded by the mesh.
 
 **Decompose Meshes**
```

Comment:
nit: It might be worth using a heading here, and then you can link to it in the text (with the `[name](#anchor)` approach)